src/compiler/scala/tools/nsc/Driver.scala

```diff
@@ -14,7 +14,8 @@ abstract class Driver {
   protected var settings: Settings = _
 
   protected def scalacError(msg: String): Unit = {
-    reporter.error(FakePos("scalac"), msg + "\n  scalac -help  gives more information")
+    val indent = "\u0020" * 2
```

Comment:
I'll use a formatting width as in my reply to adriaan.  Two spaces is too hard to verify at a glance, even in monospace.